models/library.go

```diff
@@ -53,19 +63,56 @@ type Library struct {
 	// The Path to the library in the container image. Empty string when Lockfile scan.
 	// This field is used to convert the result JSON of a `trivy image` using trivy-to-vuls.
 	FilePath string
+	Digest   string
+}
+
+func trivyInit() error {
+	if err := tlog.InitLogger(config.Conf.Debug, config.Conf.Quiet); err != nil {
+		return xerrors.Errorf("Failed to init trivy logger. err: %w", err)
+	}
+
+	javadb.Init(config.Conf.TrivyCacheDBDir, config.Conf.TrivyJavaDBRepository, config.Conf.TrivySkipJavaDBUpdate, config.Conf.Quiet, ftypes.RegistryOptions{})
+	return nil
+}
+
+var trivyInitOnce = sync.OnceValue(trivyInit)
+
+func trivyInitJavaDB() error {
+	javadb.Init(config.Conf.TrivyCacheDBDir, config.Conf.TrivyJavaDBRepository, config.Conf.TrivySkipJavaDBUpdate, config.Conf.Quiet, ftypes.RegistryOptions{})
+
+	javaDBClient, err := javadb.NewClient()
+	if err != nil {
+		return xerrors.Errorf("Failed to init trivy Java DB. err: %w", err)
+	}
+	javaDB = *javaDBClient
+	return nil
 }
 
+var trivyInitJavaDBOnce = sync.OnceValue(trivyInitJavaDB)
+
+var javaDB javadb.DB
```

Comment:
Regarding variable names, client and javaDBClient seem better than javaDB.
https://github.com/future-architect/vuls/pull/1806/files#diff-bf20e6b4202a8569d5c42386b8f6377ad10c569e8142e76825bc429f569710d4R87